app/Global.scala

```diff
@@ -1,32 +0,0 @@
-/*
- * Copyright 2014 The Guardian
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-import controllers.RepoWhitelistService
-import monitoring.SentryLogging
-import play.api._
-import play.api.mvc._
-import play.filters.csrf._
-
-object Global extends WithFilters(CSRFFilter()) with GlobalSettings {
-
-  Logger.info("java.version="+System.getProperty("java.version"))
-
-  override def onStart(app: Application) {
-    SentryLogging.init()
-    RepoWhitelistService.start()
```

Comment:
The initialisation of both these services was accidentally lost in this PR:

* without `SentryLogging`, no reporting to https://sentry.io/organizations/the-guardian/projects/prout/ would have been done
* without starting `RepoAcceptListService`, the list of 'accepted' repos would only be set at startup, and never updated until the next restart. Also, periodic logging of GitHub API consumption (which was folded into `RepoAcceptListService`) was lost.